frontend/app/src/components/AppView/styled-components.ts

```diff
@@ -124,7 +124,10 @@ export const StyledAppViewBlockContainer =
         (addPaddingForHeader && !showPadding) ||
         (isEmbedded && hasSidebar)
       ) {
-        topEmbedPadding = "3rem"
+        // Use parseFloat vs. calc to allow for JS unit test
+        topEmbedPadding = `${
+          parseFloat(theme.sizes.headerHeight) + parseFloat(theme.spacing.md)
+        }rem`
```

Comment:
Any reason why the calc could be just tested in the unit test (like `.toEqual("calc(3.5rem + 1.5rem)")`)?

It's not a big deal, but parseFloat on every render seems a little excessive where I think the rest might be of value.

Reply:
Somehow using `calc` the property doesn't show up for unit testing (it comes back as `""`...) and I wanted a test to hopefully prevent breakage in the future. I'll play with it some more though

Reply:
How about we define this as a constant outside of the component since it will always be the same value? and just use it here.

Reply:
Ah but I realize now that if the theme changes it wouldn't propagate here. :-(

Will approve. An alternative is to somehow find a way to test this with a screenshot instead of unit tests. Might be overkill though.